src/App.js

```diff
@@ -1,20 +1,28 @@
-import axios from "axios";
 import { useState } from "react";
 import { Route, Routes } from "react-router-dom";
-import "./App.css";
 import HomePage from "./components/Homepage/HomePage";
 import MoviePage from "./components/Moviepage/MoviePage";
 
 function App() {
-  const [movieInfo, setMovieInfo] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
 
   return (
-    <div className="App">
-      <Routes>
-        <Route path="/" element={<HomePage setMovieInfo={setMovieInfo} />} />
-        <Route path="movie/:id" element={<MoviePage movieInfo={movieInfo} />} />
-      </Routes>
-    </div>
+    <Routes>
+      <Route
+        path="/"
+        element={<HomePage isLoading={isLoading} setIsLoading={setIsLoading} />}
```

Comment:
Please remove isLoading and setIsLoading props. Every component should have it's own loading state.

Reply:
fixed